packages/web-components/src/checkbox/README.md

```diff
@@ -0,0 +1,121 @@
+# Checkbox
+
+> An implementation of a [checkbox](https://w3c.github.io/html-reference/input.checkbox.html) as a form-connected web-component.
+
+<br />
+
+## **Design Spec**
+
+[Link to Checkbox Design Spec in Figma](https://www.figma.com/file/1a1hBVizk7aLH76IvrnPFs/Checkbox?node-id=1666-534&t=Y5ISi4tWguXRFMdP-0)
+
+<br />
+
+## **Engineering Spec**
+
+Fluent WC3 Checkbox extends from the [FAST Checkbox](https://explore.fast.design/components/fast-checkbox) and is intended to be as close to the Fluent UI React 9 Checkbox implementation as possible. However, due to the nature of web components there will not be 100% parity between the two.
+
+<br />
+
+## Class: `Checkbox`
+
+<br />
+
+### **Component Name**
+
+`<fluent-checkbox>`
+
+<br />
+
+### **Component Template**
+
+FAST `checkboxTemplate`
+
+<br />
+
+### **Variables**
+
+| Name                    | Description          | Type                                         |
+| ----------------------- | -------------------- | -------------------------------------------- |
+| `checkboxLabelPosition` | label position types | `{ before: "before", after: "after" }`       |
+| `checkboxShape`         | checkbox shape types | `{ square: "square", circular: "circular" }` |
+| `checkboxSize`          | checkbox size types  | `{ medium: "medium", large: "large" }`       |
+
+<br />
+
+### **Fields**
+
+| Name            | Privacy | Type                    | Default                       | Description                                        |
+| --------------- | ------- | ----------------------- | ----------------------------- | -------------------------------------------------- |
+| `labelPosition` | public  | `CheckboxLabelPosition` | `CheckboxLabelPosition.after` | Indicates postion of label                         |
+| `checked`       | public  | `boolean`               | `false`                       | Indicates whether input is checked                 |
+| `indeterminate` | public  | `boolean`               | `false`                       | Indicates whether input is initially indeterminate |
```

Comment:
FUIR9 uses `checked: boolean | 'mixed'`.
The difference should be covered in **Fluent Web Component v3 v.s Fluent React 9** section of the readme